### PR TITLE
fix variant stats

### PR DESCRIPTION
**Pull request by adipaul1981**
image

### CODE REVIEW

File: src/pages/variantsSearchPage/VariantStats.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import StudyIcon from 'icons/StudyIcon';
+import ParticipantIcon from 'icons/ParticipantIcon';
+import VariantIcon from 'icons/VariantIcon';
+import OccurencesIcon from 'icons/OccurencesIcon';
+import { Card } from 'antd';
+
+const VariantStatsContainer: FC = () => {
+  let result: VariantStatsResults = useStatVariants();
+
+  if (result.loading) {
```
Comment (evans-g-crsj) [inline src/pages/variantsSearchPage/VariantStats.tsx:16]: how about using it in the Card%

File: src/pages/variantsSearchPage/VariantSearchPage.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.variant-page-space-between {
+  justify-content: space-between;
+}
+
+.variant-page-flex {
+  display: flex;
+}
+
+.variant-page-icons {
+  fill: #383f72;
```
Comment (evans-g-crsj) [inline src/pages/variantsSearchPage/VariantSearchPage.css:24]: no hardcoded colour anymore

File: src/pages/variantsSearchPage/index.tsx [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
 import { isKfInvestigator } from 'common/profile';
 import { EgoGroups } from 'store/userTypes';
+import VariantStats from './VariantStats';
+import './VariantSearchPage.css';
 
 type OwnProps = {
   egoGroups: EgoGroups;
 };
 
 const VariantPage: FC<OwnProps> = ({ egoGroups }) => (
```
Comment (evans-g-crsj) [inline src/pages/variantsSearchPage/index.tsx:14]: it's not you, but can you remove FC, pls

File: src/pages/variantsSearchPage/VariantStats.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    return null;
+  }
+
+  return (
+    <Card
+      title={
+        <div className={'variant-page-flex variant-page-space-between'}>
+          <Title level={3}>Data Release 1</Title>
+          <div>April 08th, 2021</div>
```
Comment (evans-g-crsj) [inline src/pages/variantsSearchPage/VariantStats.tsx:25]: we must can a date value from ES

**Review by evans-g-crsj — CHANGES_REQUESTED** [review]

File: src/pages/variantsSearchPage/VariantStats.tsx (hunk 4 above)
Comment (adipaul1981) [reply src/pages/variantsSearchPage/VariantStats.tsx:27]: next time

File: src/pages/variantsSearchPage/VariantStats.tsx (hunk 1 above)
Comment (adipaul1981) [reply src/pages/variantsSearchPage/VariantStats.tsx:18]: ?

**Review by evans-g-crsj — APPROVED** [review]

**Review by francisl — APPROVED** [review]
Please use const for height and width of the icons

**Pull request merged**